src/font.rs

```diff
@@ -103,18 +103,18 @@ impl Font {
     ///
     /// [v3]: http://unifiedfontobject.org/versions/ufo3/
     pub fn load<P: AsRef<Path>>(path: P) -> Result<Font, Error> {
-        Self::load_requested_data(path, DataRequest::default())
+        Self::load_requested_data(path, &DataRequest::default())
     }
 
     /// Attempt to load the requested elements of a font object from a file.
     pub fn load_requested_data(
         path: impl AsRef<Path>,
-        request: DataRequest,
+        request: impl Borrow<DataRequest>,
```

Comment:
I believe it is preferred to use `impl AsRef<T>` here, instead of `impl Borrow<T>`; `AsRef` can be implemented for more types, but `Borrow` only has a single implementor. As an example, if a method accepts `impl AsRef<Path>` it can accept `&str` or `&OsStr` in addition to `&Path`, `&PathBuf`, `PathBuf`, etc.

In this particular case they are functionally equivalent, but I'd still prefer to go with `AsRef`, if only to match the convention.


Another general question to ask about these things is whether or not there is a clear benefit. There are two potential downsides: the first is that this makes for slightly more awkward type signatures, and the second is that this could increase code size, if the caller ends up passing multiple concrete types, e.g. if they happen to use both `&DataRequest` and `DataRequest` in the same code base.

My hunch is that the second case is unlikely; I would expect the caller to use one or the other, and likely to only use this API in a single place anyway, so I think that this is fine. My own inclination would be to probably just accept `&DataRequest` here, but I think `AsRef<DataRequest>` is fine. In general, though, I would save the `AsRef` signature for places where it is likely to provide a clear benefit; a good example is `AsRef<Path>`, which allows the user to pass in some string type, such as from a command-line argument.

Reply:
Thanks Colin!

> Another general question to ask about these things is whether or not there is a clear benefit. 

Concerns noted.  I don't feel strongly about this change.  In my mind, it was simply to avoid breakage for anyone who has existing code that is based on an owned type parameter.  

> My own inclination would be to probably just accept &DataRequest here,

Happy to change this

Reply:
> Concerns noted. I don't feel strongly about this change. In my mind, it was simply to avoid breakage for anyone who has existing code that is based on the previous defaults.

I think this is good rationale! I'd be happy with either `&DataRequest` or `AsRef<DataRequest>`.